### PR TITLE
Peer Review

### PR DESCRIPTION
**Pull request by PPableo**
Peer Review - Juno College - Greg/Kwame

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,41 @@
-import './App.css';
+import "./App.scss"
```
Comment (GregCrutchlow) [inline src/App.js:1]: Using Sass, nice. Make sure to add those ';' after your lines of code! Not really necessary.

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,41 @@
...
+import { useState } from 'react';
+import { Routes, Route, Link } from "react-router-dom";
+import Header from './components/Header';
+// import ProductDisplay from './components/ProductDisplay';
+import Footer from './components/Footer';
+import Service from './components/main/Service';
+import LandingPageImg from "./components/LandingPageImg";
+
+// <div className="App">
```
Comment (GregCrutchlow) [inline src/App.js:11]: Make sure to remove all commented out code, or place at the bottom.

File: src/App.scss [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    border: none;
+    outline: none;
+    background-color: none;
+}
+
+.row {
+    width: 50%;
+}
+
+// .wrapper {
```
Comment (GregCrutchlow) [inline src/App.scss]: Removing comments again.

File: src/App.scss [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    display: flex;
+    flex-shrink: 0;
+    min-height: 3.3rem;
+}
+
+/* --------------------------------- SEARCH --------------------------------- */
+
+.searchBox {
+    width: 80%;
+    border: 2px solid rgba(0,0,0, 0.1);
```
Comment (GregCrutchlow) [inline src/App.scss]: Using a Alpha channel is good, you could use a converter to put that into hex, but I don't think its necessary.

File: src/components/Footer.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+const Footer = () => {
+
+    // console.log("Footer has rendered")
+
+    return (
+        <div className="footer">
+            <h4>"Created at Juno College"</h4>
```
Comment (GregCrutchlow) [inline src/components/Footer.js:4]: Make sure to add your name as well for your own benefit.

File: src/components/Header.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+const Header = () => {
+
+    // console.log('Header has rendered');
+
+    return (
+        <>
+            <Head />
+            <Search />
+            <Navbar />
+            {/* <LandingPageImg /> */}
```
Comment (GregCrutchlow) [inline src/components/Header.js]: Remove comment.

File: src/components/Navbar.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
+import { Link } from 'react-router-dom';
```
Comment (GregCrutchlow) [inline src/components/Navbar.js:1]: Good implementation of what we learned yesterday.

File: src/components/main/ProductDisplay.js [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+                name: newState.name,
+                price: newState.price
+            };
+
+            for (let key in data) {
+                newState.push(data[key])
+            }
+            // console.log(newState);
+
+            // console.log(newState)
```
Comment (GregCrutchlow) [inline src/components/main/ProductDisplay.js]: Just remove commented code.

File: src/components/main/ProductDisplay.js [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+            // console.log(response.val)
+            console.log(newState)
+        });
+    }, []);
+    return (
+        <div className="products">
+            <ul>
+                {services.map((service) => {
+                    return (
+                        // <li key={service.name}>
```
Comment (GregCrutchlow) [inline src/components/main/ProductDisplay.js]: Remove commented code.

**Review by GregCrutchlow — COMMENTED** [review]
Good code, I like the implementation of Sass, just make sure to remove commented code and change pseudo code to make what you're doing on page.

File: src/App.scss [hunk 10]
```diff
@@ -0,0 +1,241 @@
...
+
+/* ---------------------------------- SETUP --------------------------------- */
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+/* --------------------------------- GENERAL -------------------------------- */
```
Comment (kwametsunami) [inline src/App.scss]: I like your organization!

File: src/components/Header.js [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+        </>
```
Comment (kwametsunami) [inline src/components/Header.js]: good use of fragments

File: src/components/Navbar.js [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+                                <Link to="./">HOME</Link>
+                            </li>
+                            <li>
+                                <Link to="./">SERVICES</Link>
+                            </li>
+                            <li>
+                                <Link to="./store">store</Link>
+                            </li>
+                            <li>
+                                <Link to="./contact">contact</Link>
```
Comment (kwametsunami) [inline src/components/Navbar.js:26]: awesome link work! not sure if it's intentional, but services doesn't like to anything

File: src/components/main/CategoriesModal.js [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+            <div className="category">
+                {data.map((value, index) => {
+                     return(
+                    <div className="box fFlex" key={index}>
+                        <img src={value.cateImg} alt="category image" />
+                        <span>{value.cateName}</span>
+                    </div>
+                )
+                }
+                )
```
Comment (kwametsunami) [inline src/components/main/CategoriesModal.js]: indentation on the brackets

**Review by kwametsunami — COMMENTED** [review]
great styling and use of sass! your components are well organized and are setup to add more features and work on in the future

**Pull request merged**